universe/marathon.json.mustache

```diff
@@ -31,7 +31,8 @@
       "JVM_OPTS": "{{advanced.jvm-opts}}",
       "JENKINS_OPTS": "{{advanced.jenkins-opts}}",
       "PROMETHEUS_ENDPOINT": "{{advanced.prometheus-endpoint}}",
-      "SSH_KNOWN_HOSTS": "{{networking.known-hosts}}"
+      "SSH_KNOWN_HOSTS": "{{networking.known-hosts}}",
+      "MARATHON_NAME": "marathon"
```

Comment:
Should this be hardcoded here or added to the config.json for the Universe package?

Reply:
We want to hardcode it, since almost no customer will ever alter it, and those that do change it are deploying to a MoM anyway. Since customers deploying to a MoM do package_describe and modify things anyway, having this be hardcoded here is easy for them to change while not confusing 99% of customers.